### ISSUE TITLE
[BUG] All links to official plugin demos are broken

### ISSUE THREAD

**Issue by ibnishak** [TiddlyWiki/TiddlyWiki5#6319]
How to replicate

1. Go to tiddlywiki.com
2. Click on save button and download a copy of tiddlywiki.com
3. Open the locally saved copy in any browser
4. Open `Markdown plugin` tiddler and click the link to demo

Reason: Pretty link syntax of the links are wrong

**Comment by pmario** [comment]
You are right, the prettylinks seem to be wrong: It should be `See [ext[plugins/tiddlywiki/markdown|https://tiddlywiki.com/plugins/tiddlywiki/markdown]]` but it is switched atm.

**Comment by Jermolene** [comment]
> You are right, the prettylinks seem to be wrong: It should be `See [ext[plugins/tiddlywiki/markdown|https://tiddlywiki.com/plugins/tiddlywiki/markdown]]` but it is switched atm.

Sorry @pmario that is not correct. The current formatting of these links is intentional. Please see my comment over in #6318.

**Comment by pmario** [comment]
I've just read the comment. ... OK. ... And I think we discussed this some time ago already. ... I've forgotten about that ... I'd really love to get that fixed.
